src/applications/vaos/new-appointment/components/ReviewPage/AppointmentDate.jsx

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
 import moment from 'moment';
-import { getTimezoneAbbrByFacilityId } from '../../../utils/timezone';
+import {
+  getTimezoneAbbrByFacilityId,
+  getTimezoneByFacilityId,
+} from '../../../utils/timezone';
 
 export default function AppointmentDate(props) {
+  const timezone = getTimezoneByFacilityId(props.facilityId);
+  if (props.dates[0].endsWith('Z') && timezone) {
+    return props.dates?.map((selected, i) => (
```

Comment:
We don't seem to have a test that covers the code in this if statement, it would be good to add one. I think you could update the tests in `VAOS <ReviewPage> direct scheduling with v2 api` to have slot data in UTC and then update one of the tests to check to make sure the time is correct.